src/cc_catalog_airflow/dags/provider_api_scripts/brooklyn_museum.py

```diff
@@ -0,0 +1,203 @@
+"""
+Content Provider:       Brooklyn Museum
+
+ETL Process:            Use the API to identify all CC licensed artworks.
+
+Output:                 TSV file containing the images and respective meta-data.
+
+Notes:                  https://www.brooklynmuseum.org/opencollection/api
+						3000 calls per day (per API Key)
+"""
+import logging
+import time
+import os
+from common.requester import DelayedRequester
+from common.storage import image
+
+logger = logging.getLogger(__name__)
+
+DELAY       = 3.0 #time delay (in seconds)
```

Comment:
Please use `pycodestyle` to check for PEP-8 compliance and fix all PEP-8 violations.

Reply:
sure. Thanks.